src/linter/ui5Types/bestPractices.ts

```diff
@@ -314,29 +323,48 @@ function reportResults(
 	if (hasAsyncInterface !== AsyncInterfaceStatus.true) {
 		if ([AsyncInterfaceStatus.propNotSet, AsyncInterfaceStatus.false].includes(rootViewAsyncFlag) ||
 			[AsyncInterfaceStatus.propNotSet, AsyncInterfaceStatus.false].includes(routingAsyncFlag)) {
-			const componentsToAddress = [];
+			let message = "Root View and Routing are not configured to load their modules asynchronously.";
+			let messageDetails = "{@link topic:676b636446c94eada183b1218a824717 Use Asynchronous Loading}. " +
+				"Implement sap.ui.core.IAsyncContentCreation interface in Component.js or set async flags for " +
+				"\"sap.ui5/routing/config\" and \"sap.ui5/rootView\" in the manifest";
+
 			if (AsyncInterfaceStatus.parentPropNotSet !== rootViewAsyncFlag) {
-				componentsToAddress.push("Root View");
-			}
-			if (AsyncInterfaceStatus.parentPropNotSet !== routingAsyncFlag) {
-				componentsToAddress.push("Routing");
+				// sap.ui5/rootView is not set at all, so skip it in the message
+				message = "Routing is not configured to load its targets asynchronously.";
+				messageDetails = "{@link topic:676b636446c94eada183b1218a824717 Use Asynchronous Loading}. " +
+				"Implement sap.ui.core.IAsyncContentCreation interface in Component.js or set async flag for " +
+				"\"sap.ui5/routing/config\" in the manifest.";
```

Comment:
Are there any tests that cover this behavior? I've tested this with an application and am getting the exact opposite from what I would expect. I have a rootView defined without async flag, and the message complains about a missing flag in the routing. In the Positive_1 fixture both flags are missing, but there's only a message for missing routing flag.

Reply:
Thanks!
Fixed it with https://github.com/SAP/ui5-linter/pull/73/commits/417a94b0ba2dc8554093c0668c2ca14f7d7ed3d2
Also checked the snapshots and now everything seems to be correct. 

Everything case I imagined is covered now. If you can think of some other, please let me know

Reply:
I still see no test case that covers those two messages. Coverage also shows them as not covered. But maybe I'm missing something here. I'm on https://github.com/SAP/ui5-linter/commit/417a94b0ba2dc8554093c0668c2ca14f7d7ed3d2.

Reply:
Please, have a look at `Positive_2/Component.js`: https://github.com/SAP/ui5-linter/pull/73/files#diff-0ec5155380fd3031aee943bec0ef111257c570eff2d4ceef2cf631ece3c28342R168

Reply:
It has different messages. My comment here is about `Routing is not configured [...]` and `Root View is not configured [...]`.

Reply:
Ah, thanks a lot! I've missed it.

I have added the cases here https://github.com/SAP/ui5-linter/pull/73/commits/604ef2257de05ff4c0fb16d9290d9772ea5218a1